src/agent.spec.ts

```diff
@@ -1,74 +1,173 @@
 import {
-  FindingType,
-  FindingSeverity,
   Finding,
-  HandleTransaction,
+  FindingSeverity,
+  FindingType,
   createTransactionEvent,
-  ethers,
+  HandleTransaction,
+  TransactionEvent,
+  
 } from "forta-agent";
-import agent, {
-  ERC20_TRANSFER_EVENT,
-  TETHER_ADDRESS,
-  TETHER_DECIMALS,
-} from "./agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import provideHandleTransaction from "./agent";
```

Comment:
```suggestion
import { provideHandleTransaction } from "./agent";
```
Once we export the object from `agent.ts`, this should be updated.